src/treeView/repositories/repositories.ts

```diff
@@ -24,8 +27,8 @@ export function activateTreeViewRepositories() {
     vscode.env.clipboard.writeText(`${repo.url}.git`));
 
   // Reload repos
-  vscode.commands.registerCommand('githubRepoMgr.commands.repos.reload', () =>
-    repositories.loadRepos());
+  // vscode.commands.registerCommand('githubRepoMgr.commands.repos.reload', () =>
```

Comment:
Probably should confirm with user before doing this as its time consuming action

Reply:
Nice progress! I personally wouldn't like having to confirm the reload. The time it takes for me is so low that having to confirm wouldn't be worthy. Maybe add a setting for it (default false, as I think most of the users don't have lots of repos hehe)

Reply:
Have you considered adding analytics to the plugin so you can confirm whether this in fact true assumption? like just have a log somewhere of how many repos someone has

Reply:
Never thought about that, it's a creative idea. However, I really don't think usage info would have enough benefits. I also don't like anything that gets my data (I always disable those analytics stuff in every site/program that uses or requests to use), so, also wouldn't like to have analytics in the extension. And, as this extension handles some sensitive data, even being tokens, I think people would feel more insecure with the info of analytics getting some user info.

I know VsCode itself have a lot of analytics in its code, but, it is more trustful than a barely known extension